src/main/java/roomescape/common/exception/PastDateException.java

```diff
@@ -0,0 +1,7 @@
+package roomescape.common.exception;
+
+public class PastDateException extends RuntimeException {
+    public PastDateException(String message) {
+        super(message);
```

Comment:
클래스명에서 예외의 성격이 명확하게 드러나므로, 
예외 메세지를 아래와 같이 통일하는 것도 방법이겠네요 😄 

```suggestion
        super("과거 시간은 예약 등록을 할 수 없습니다. date = " + date);
```

Reply:
어차피, 해당 예외 클래스는 특정 예외 메시지만 보여주게 될 것이니까, 
예외 메시지를 하나로 관리할 수 있도록 하는 방법이군용!

좋은 것 같아요!